daemon/src/auto_rollover.rs

```diff
@@ -70,39 +69,40 @@ where
             .expect("actor to be able to give address to itself");
 
         for id in cfd_ids {
-            let cfd = try_continue!(load_cfd(id, &mut conn).await);
+            try_continue!(
+                async {
+                    let cfd = load_cfd(id, &mut conn).await?;
+                    cfd.can_auto_rollover_taker(OffsetDateTime::now_utc())?;
 
-            if let Err(e) = cfd.can_auto_rollover_taker(OffsetDateTime::now_utc()) {
-                tracing::trace!(%id, "Cannot roll over: {:#}", e);
-                continue;
-            }
+                    this.send(Rollover(id)).await??;
 
-            this.send_async_safe(Rollover { id }).await?;
+                    anyhow::Ok(())
+                }
+                .await
```

Comment:
🔧 Let's add the "Cannot roll over" context to the result of this async block, so that any of the errors which could prevent the roll over have the appropriate context.